runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -186,9 +173,9 @@ def test_infer_mixed_keys(device):
     tensor2 = Tensor(data2)
 
     request = exec_net.create_infer_request()
-    with pytest.raises(TypeError) as e:
+    with pytest.raises(KeyError) as e:
```

Comment:
this test was about mixed keys, so as user now can pass mixed input dict check that it works